src/Components/Address/useAddressAutocomplete.ts

```diff
@@ -45,10 +45,6 @@ export const useAddressAutocomplete = (
   const isAddressAutocompleteEnabled =
     isAPIKeyPresent && isFeatureFlagEnabled && isUSAddress
 
-  // // TODO: Remove this console log after testing
-  // console.log({ isAPIKeyPresent, isFeatureFlagEnabled, isUSAddress })
-
-  // reset suggestions if the country changes
```

Comment:
I think that it would be OK to keep this comment about resetting the suggestions.

Reply:
sure, I'll add it back in